packages/PyTrilinos/src/Teuchos_ParameterList.i

```diff
@@ -472,6 +472,19 @@ Teuchos::ParameterList::values
     return NULL;
   }
 
+  /******************************************************************/
+  // String constructor (required because in C++, this constructor has
+  // an optional ParameterListModifier, and we are ignoring
+  // ParameterListModifiers
+  ParameterList(const string & name)
+  {
+    Teuchos::ParameterList * plist = new Teuchos::ParameterList(name);
+    if (plist == NULL) goto fail;
```

Comment:
Would you consider preferring `nullptr` to `NULL` in new code?

Reply:
Sure.  I could make a universal change, as well.